webapp/components/edit_channel_header_modal.jsx

```diff
@@ -81,15 +90,19 @@ class EditChannelHeaderModal extends React.Component {
     }
 
     onShow() {
-        const textarea = ReactDOM.findDOMNode(this.refs.textarea);
-        Utils.placeCaretAtEnd(textarea);
         this.submitted = false;
     }
 
     onHide() {
         this.setState({show: false});
     }
 
+    focusTextbox(keepFocus = false) {
```

Comment:
Can you remove the `keepFocus` parameter since it's never being passed into by any code that calls `focusTextbox`?